.travis.yml

```diff
@@ -17,85 +12,79 @@ before_install:
 install:
   -
 
-env:
-  -JDK=Java8
-  -JDK=Java9
-  -JDK=Java10
-  -JDK=Java11-EA
-  -JDK=Java12-EA
-
-matrix:
+jobs:
   fast_finish: true
   include:
     - jdk: oraclejdk8
       env:
         - DESC="acceptance tests"
-        - JDK=Java8
         - CMD="mvn install --projects acceptance-tests --also-make --activate-profiles all"
 
     - jdk: oraclejdk8
       env:
         - DESC="findbugs"
-        - JDK=Java8
         - CMD="mvn install findbugs:check --projects '!scala-unit-tests,!jmh-scala-tests,!jmh-tests,!jcstress-tests,!p2-repository' --activate-profiles all -DskipTests=true"
+
     - jdk: oraclejdk8
       env:
         - DESC="checkstyle"
-        - JDK=Java8
         - CMD="mvn install checkstyle:check --activate-profiles all -DskipTests=true"
 
     - jdk: oraclejdk8
       env:
         - DESC="unit tests"
-        - JDK=Java8
         - CMD="mvn install"
 
     - jdk: oraclejdk8
       env:
         - DESC="compile performance-tests"
-        - JDK=Java8
         - CMD="mvn install --projects performance-tests --also-make --activate-profiles all -DskipTests=true"
 
     - jdk: oraclejdk9
       env:
-        - DESC="unit tests Java 9"
-        - JDK=Java9
+        - DESC="unit tests oraclejdk9"
         - CMD="mvn install"
 
-    - jdk: oraclejdk8
+    - jdk: openjdk10
+      env:
+        - DESC="unit tests openjdk10"
+        - CMD="mvn install"
+
+    - jdk: oraclejdk10
       env:
-        - DESC="Java 10 regression"
-        - JDK=Java10
-        - CMD=""
+        - DESC="unit tests oraclejdk10"
+        - CMD="mvn install"
+
+    - jdk: openjdk11
+      if: type = cron
```

Comment:
Adding the condition `if: type = cron` means these builds will never run or even show up except on a schedule. Since we only run these builds for information and not to block commits, I think it makes sense to run them nightly. This should make builds faster and reduce confusion for contributors.

Reply:
I disagree with this. I would rather have it run for every PR. It helps us isolate issues in the PRs. With a cron job we will definitely miss investigations

Reply:
Another option is `if: type = push AND fork = false`. I just had trouble getting it to work properly. Then it should happen on every new change, but contributors would never see it. I figured cron is good enough because our pace of change isn't that fast, so daily would catch almost everything. And my top priorities are not making contributors wait, or see failing builds they don't need to worry about.

WDYT?

Reply:
This is fair. 
I do need to check the policy of early adopters (since Eclipse Collections is a part of it) if they can be fine running it at a frequency desired by the project. It is up to us to decide then we can change it to a cron interval of once every day.

Reply:
Actually I am debating this. If there are build failures due to issues with URLs we can always explain it to the contributors. But in case there are actual issues with the code, they should be fixed. I think we should keep it for all PRs. Latest example of my own PR 😄 : https://travis-ci.org/eclipse/eclipse-collections/builds/427021454